lib/shared/screens/admin/shared/components/content-page-builder/index.js

```diff
@@ -90,6 +90,12 @@ export default class ContentPageBuilderContainer extends Component {
     confirmUnpublish: PropTypes.bool.isRequired
   };
 
+  static defaultProps = {
+    publish: () => {},
+    unpublish: () => {},
```

Comment:
So should I just remove the required flag on PropTypes for these functions? It seems there are scenarios where they aren't always passed in because they aren't necessary.  I will take a look and see if that is an issue.  But from what I saw the other day this was the case.